docs/cromwell_features/RetryWithMoreMemory.md

```diff
@@ -0,0 +1,35 @@
+#Retry with More Memory
+
+With this feature one can specify an array of strings which when encountered in the `stderr` file by Cromwell, 
+allows the task to be retried with more memory. The retry will be counted against the `maxRetries` count mentioned in 
+the `runtimeAtrributes` in the task. There are 2 settings for this feature:
+* `system.memory-retry-error-keys` : the error keys that need to be set in Cromwell config
+* `memory_retry_multiplier` : [optional] the factor by which the memory should be multiplied while retrying. This needs 
+to be passed in through workflow options and should be in the range `1.0 <= multiplier <= 99.0` (note: if set to `1.0` the task
```

Comment:
This may be the most seagull comment ever (option + `<` on Mac = `≤`)
```suggestion
to be passed in through workflow options and should be in the range `1.0 ≤ multiplier ≤ 99.0` (note: if set to `1.0` the task
```

Reply:
TIL. This is pretty cool. I have replaced `<` with `≤` 👍